### PR TITLE
add quayLogin master integration

### PR DESCRIPTION
**Pull request by amalfra**
Fixes https://github.com/Shippable/admiral/issues/1146

### CODE REVIEW

File: common/scripts/configs/master_integration_fields.sql [hunk 1]
```diff
@@ -869,6 +869,32 @@ do $$
...
+    end if;
+
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 235) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (235, '580ee981a337bd12008fc43f', 'username', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-04', '2017-10-04');
+    end if;
+
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 236) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (236, '580ee981a337bd12008fc43f', 'password', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-04', '2017-10-04');
```
Comment (shrivara) [inline common/scripts/configs/master_integration_fields.sql]: Isn't this a secure field?

File: common/scripts/configs/master_integration_fields.sql [hunk 2]
```diff
@@ -869,6 +869,32 @@ do $$
...
+    end if;
+
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 237) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (237, '580ee981a337bd12008fc43f', 'email', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-04', '2017-10-04');
+    end if;
+
+    if not exists (select 1 from "masterIntegrationFields" where "id" = 238) then
+      insert into "masterIntegrationFields" ("id", "masterIntegrationId", "name", "dataType", "isRequired", "isSecure","createdBy", "updatedBy", "createdAt", "updatedAt")
+      values (238, '580ee981a337bd12008fc43f', 'accessToken', 'string', true, false,'54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2017-10-04', '2017-10-04');
```
Comment (shrivara) [inline common/scripts/configs/master_integration_fields.sql]: Also, this.

**Comment by shrivara** [comment]
@amalfra, `accessToken` doesn't seem to be a secure field in the older integration. Could you confirm whether this should secure or not? The name suggests that it should be.

**Comment by amalfra** [comment]
@shrivara I have changed ```accessToken``` to not secure as in old old integration. As discussed with @rageshkrishna we are not changing any fields and hence replicating them as it is in old integration. Didn't find where the accessToken is used since the docker login command will use username and password only

**Comment by shrivara** [comment]
Okay, merging.

**Pull request merged**